docs/admin/authorization.md

```diff
@@ -33,30 +33,36 @@ If multiple modes are provided the set is unioned, and only a single authorizer
 
 will always allow.
 
-## ABAC Mode
-
-### Request Attributes
+## Request Attributes
 
 A request has the following attributes that can be considered for authorization:
 
   - user (the user-string which a user was authenticated as).
   - group (the list of group names the authenticated user is a member of).
+  - "extra" (a map of arbitrary string keys to string values, provided by the authentication layer)
   - whether the request is for an API resource.
   - the request path.
-    - allows authorizing access to miscellaneous endpoints like `/api` or
-`/healthz` (see [kubectl](#kubectl)).
+    - allows authorizing access to miscellaneous non-resource endpoints like `/api` or `/healthz` (see [kubectl](#kubectl)).
   - the request verb.
-    - API verbs like `get`, `list`, `create`, `update`, `watch`, `delete`, and
-`deletecollection` are used for API requests
-    - HTTP verbs like `get`, `post`, `put`, and `delete` are used for non-API
-requests
-  - what resource is being accessed (for API requests only)
-  - the namespace of the object being accessed (for namespaced API requests
-only)
-  - the API group being accessed (for API requests only)
-
-We anticipate adding more attributes to allow finer grained access control and
-to assist in policy management.
+    - API verbs `get`, `list`, `create`, `update`, `watch`, `delete`, and `deletecollection` are used for resource requests
```

Comment:
Yes, "patch" should be in the list. I'll add it. You're right about why I removed "like." Since there is no other reference docs on authz, it didn't make sense to me to provide an incomplete list without a place the user could go to find out the complete list.